src/main/scala/fr/acinq/bitcoin/scalacompat/Script.scala

```diff
@@ -158,6 +164,8 @@ object Script {
 
   def isPay2wpkh(script: Seq[ScriptElt]): Boolean = bitcoin.Script.isPay2wsh(script.map(scala2kmp).asJava)
 
+  def pay2tr(publicKey: XonlyPublicKey): Seq[ScriptElt] = bitcoin.Script.pay2tr(publicKey.pub).asScala.map(kmp2scala).toList
```

Comment:
Wouldn't it be useful to also define a `witnessPay2tr(publicKey: XonlyPublicKey, sig: ByteVector64): ScriptWitness` for the keypath spend case? Maybe it should be named explicitly `witnessPay2trKeyPath`

Reply:
I think it's a good idea, we should use what people are most used to in the ecosystem. But that can wait for a future PR where we add better abstractions for taproot overall.